README.md

```diff
@@ -255,17 +237,15 @@ The default configuration can be found in `config/default.json`.
 }
 ```
 
-The default configuration is merged with (and overridden by) environment-specific configuration that can be specified at startup with the [`NODE_ENV` environment variable](#node-env).
-
-If you want to change your local configuration, we suggest you create a `config/development.json` file with overridden values.
+The default configuration is merged with (and overridden by) environment-specific configuration that can be specified at startup with the `NODE_ENV` environment variable. For example, you would run `NODE_ENV=development npm start` to load the `development.json` configuration file.
```

Comment:
This example is not so good. Because if it's defined, `development.json` will already be loaded when running `npm start` even if no `NODE_ENV` are specified.